src/ActionList/List.tsx

```diff
@@ -10,7 +10,9 @@ import {hasActiveDescendantAttribute} from '../behaviors/focusZone'
 
 type RenderItemFn = (props: ItemProps) => React.ReactElement
 
-export type ItemInput = ItemProps | ((Partial<ItemProps> & {renderItem: RenderItemFn}) & {key?: Key})
+export type ItemInput =
```

Comment:
Since `ActionList`’s `items` type now diverges from `ItemProps`, does the [props documentation for `ActionList`](https://primer.style/react/ActionList#props) need to be updated?

Reply:
Good catch, thanks!